com.unity.render-pipelines.universal/Shaders/PostProcessing/UberPost.shader

```diff
@@ -9,6 +9,7 @@ Shader "Hidden/Universal Render Pipeline/UberPost"
         #pragma multi_compile_local _ _FILM_GRAIN
         #pragma multi_compile_local _ _DITHERING
 		#pragma multi_compile_local _ _LINEAR_TO_SRGB_CONVERSION
+        #pragma multi_compile _ _USE_DRAW_PROCEDURAL
```

Comment:
Same here: adding more "multi_compile _ " will only increase shader build times. For fill-rate intensive postprocessing, use quad mesh is sufficient.